app/screens/send-bitcoin-screen/destination-information.tsx

```diff
@@ -6,21 +6,22 @@ import { useAppConfig } from "@app/hooks"
 import React from "react"
 import { Text, View } from "react-native"
 import EStyleSheet from "react-native-extended-stylesheet"
-import { SendBitcoinDestinationState } from "./send-bitcoin-reducer"
+import { DestinationState, SendBitcoinDestinationState } from "./send-bitcoin-reducer"
 import { IntraledgerPaymentDestination } from "@galoymoney/client/dist/parsing-v2"
+import { InvalidDestinationReason } from "./payment-destination/index.types"
 
 const toLnAddress = (handle: string, lnDomain: string) => {
   return `${handle}@${lnDomain}`
 }
```

Comment:
lnDomain doesn't change. this could be memoized, such that when it's called only the handle needs to be passed 

```suggestion
const toLnAddress = (lnDomain: string)(handle: string)  => {
  return `${handle}@${lnDomain}`
}
```

Reply:
means `toLnAddress ` should be in `destinationStateToInformation`